client/src/main/java/org/spine3/base/Commands.java

```diff
@@ -251,4 +250,44 @@ public static boolean isScheduled(Command command) {
         }
         return false;
     }
+
+    /**
+     * Sets a new scheduling time to {@link CommandContext.Schedule}.
```

Comment:
What is ‘scheduling time’? Is it the time after which the command should be executed? Or, is it the time when the command was put into the scheduled command list?